v0/destinations/am/transform.js

```diff
@@ -377,6 +384,14 @@ function responseBuilderSimple(
       // ====================
       // fixVersion(payload, message);
 
+      if (payload.user_properties) {
```

Comment:
Similarly is it possible to filter out these `user_properties` when we are mapping them instead deleting later?

Reply:
Same for user_properties as well. We are passing everything we are getting in traits. So therefore deleting unwanted fields at later stage.